cmd/search.go

```diff
@@ -82,7 +85,7 @@ please see https://pkg.go.dev/github.com/zmb3/spotify?tab=doc#Client.Search`,
 			regexID := regexp.MustCompile(`(spotify:track:)(.*)`)
 			trackID := spotify.ID(regexID.FindStringSubmatch(string(toPlay))[2])
 			seeds := spotify.Seeds{Tracks: []spotify.ID{trackID}}
-			recommends, err := conf.Client.GetRecommendations(seeds, nil, nil)
+			recommends, err := conf.Client.GetRecommendations(ctx, seeds, nil, nil)
```

Comment:
I'm not sure why this errors, but for some reason, when searching a track, we get a runtime panic pointing to this line.

Reply:
Ah, so it looks like the fix for this is to remove the last `nil` in the call to `client.GetRecommendations` here.

Reply:
Could you remove the last `nil` in the call to `client.GetRecommendations` here?